config/v1/types_cluster_operator.go

```diff
@@ -158,12 +158,15 @@ const (
 	OperatorAvailable ClusterStatusConditionType = "Available"
 
 	// OperatorProgressing indicates that the component (operator and all configured operands)
-	// is actively rolling out new code, propagating config changes, or otherwise
+	// is actively rolling out new code, propagating config changes (e.g, a version change), or otherwise
 	// moving from one steady state to another. Operators should not report
 	// progressing when they are reconciling (without action) a previously known
 	// state. If the observed cluster state has changed and the component is
 	// reacting to it (scaling up for instance), Progressing should become true
 	// since it is moving from one steady state to another.
```

Comment:
Did you want to crisp up this `scaling up for instance` sentence, while you're here?  Some components currently go `Progressing=True` when a DaemonSet launches new Pods in response to Node scaling.  That's "the component is reacting to it", although it's not "the cluster operator controller is reacting" directly, it's down at the "the DaemonSet controller is reacting" level.